conda/_vendor/boltons/setutils.py

```diff
@@ -17,10 +14,7 @@
 
 from bisect import bisect_left
 from itertools import chain, islice
-try:
-    from collections.abc import MutableSet
-except ImportError:
-    from collections import MutableSet
+from collections import MutableSet
```

Comment:
this does not work in Python 3.10; it looks like that has been fixed in the newest revision.